modules/overview-of-distributed-workloads.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: CONCEPT
+
+[id='overview-of-distributed-workloads_{context}']
+= Overview of distributed workloads
+
+[role='_abstract']
+You can use the distributed workloads feature to queue, scale, and manage the resources required to run data science workloads across multiple nodes in an OpenShift cluster simultaneously. Data science workloads include several types of artificial intelligence (AI) workloads, including machine learning (ML) and Python workloads.
+
+Distributed workloads provide the following benefits:
+
+* You can iterate faster and experiment more frequently because of the reduced processing time.
+* You can use larger datasets, which can lead to more accurate models.
+* You can use complex models that could not be trained on a single node.
+
+The distributed workloads infrastructure includes the following components:
+
+* *CodeFlare SDK:* Defines and controls the remote distributed compute jobs and infrastructure for any Python-based environment
```

Comment:
Not a big thing, but I like to use a description list rather than bullets in these cases. The formatting is a little more effective. See https://docs.asciidoctor.org/asciidoc/latest/lists/description/.